packages/desktop-gui/src/prompts/prompts.jsx

```diff
@@ -0,0 +1,362 @@
+import React, { Component, useState } from 'react'
```

Comment:
Seems like this file could be refactored into more managable smaller files. Right now the purpose of the file is a bit broad covering ALL prompts content and the prompt window functionality itself.

Reply:
Yeah I agree with this - I don't think its blocking for this PR so I'll go ahead and do some refactoring when we add a third prompt next week